chia/full_node/coin_store.py

```diff
@@ -22,6 +22,28 @@
 log = logging.getLogger(__name__)
 
 
+async def add_coin_records_to_db(db_wrapper: DBWrapper2, records: list[CoinRecord]) -> None:
```

Comment:
I think most of the value of this change is gained by moving this function into a file that lives under `chia/_tests/` to remove it from production code and turn it into test code